.github/workflows/sycl_cleanup.yml

```diff
@@ -3,8 +3,8 @@ name: Cleanup
 on:
   workflow_run:
     workflows:
-    - SYCL
-    - SYCL Nightly Builds
+    - SYCL Pre Commit
+    - SYCL Nightly
     - SYCL Post Commit
```

Comment:
I'm not sure about this syntax, but it looks like workflow names are used here. If so, these should be aligned with the names in the corresponding workflow files.